compute_tools/src/disk_quota.rs

```diff
@@ -0,0 +1,25 @@
+use anyhow::Context;
+
+pub const DISK_QUOTA_BIN: &str = "/neonvm/bin/set-disk-quota";
+
+/// If size_bytes is 0, it disables the quota. Otherwise, it sets filesystem quota to size_bytes.
+/// `fs_mountpoint` should point to the mountpoint of the filesystem where the quota should be set.
+pub fn set_disk_quota(size_bytes: u64, fs_mountpoint: &str) -> anyhow::Result<()> {
```

Comment:
Is it possible to not return an anyhow::result?

Reply:
It is, but anyhow::Result is very convenient for `.context()` here. Do you have a suggestion for something else?

Reply:
I typically just find that we seem to abuse anyhow::Result, but not worth blocking the PR for.